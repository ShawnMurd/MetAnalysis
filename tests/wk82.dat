  N= 180
   Pressure (mb)     Temperature (C)     Mixing Ratio (g/kg)
   -------------     ---------------     -------------------
994.4     26.4     14.00
983.2     25.6     14.00
972.1     24.8     14.00
961.1     24.0     14.00
950.1     23.2     14.00
939.3     22.4     14.00
928.6     21.7     14.00
918.0     20.9     14.00
907.4     20.2     14.00
897.0     19.4     14.00
886.7     18.7     14.00
876.4     18.0     14.00
866.3     17.2     13.81
856.2     16.5     13.27
846.2     15.8     12.76
836.3     15.1     12.26
826.5     14.4     11.78
816.8     13.7     11.32
807.2     13.0     10.87
797.7     12.3     10.45
788.2     11.6     10.03
778.8     10.9     9.63
769.6     10.2     9.25
760.4     9.5     8.88
751.3     8.8     8.52
742.3     8.1     8.18
733.3     7.4     7.84
724.5     6.7     7.52
715.7     6.0     7.22
707.0     5.3     6.92
698.4     4.6     6.63
689.9     4.0     6.36
681.5     3.3     6.09
673.1     2.6     5.83
664.8     1.9     5.59
656.7     1.2     5.35
648.5     0.6     5.12
640.5     -0.1     4.90
632.6     -0.8     4.69
624.7     -1.5     4.48
616.9     -2.2     4.28
609.2     -2.8     4.10
601.5     -3.5     3.91
593.9     -4.2     3.74
586.5     -4.8     3.57
579.0     -5.5     3.41
571.7     -6.2     3.25
564.4     -6.9     3.10
557.2     -7.5     2.96
550.1     -8.2     2.82
543.1     -8.9     2.69
536.1     -9.6     2.56
529.2     -10.2     2.44
522.4     -10.9     2.32
515.6     -11.6     2.21
508.9     -12.2     2.10
502.3     -12.9     2.00
495.7     -13.6     1.90
489.2     -14.2     1.81
482.8     -14.9     1.72
476.5     -15.6     1.63
470.2     -16.2     1.55
464.0     -16.9     1.47
457.8     -17.6     1.39
451.7     -18.3     1.32
445.7     -18.9     1.25
439.8     -19.6     1.19
433.9     -20.3     1.12
428.0     -20.9     1.06
422.3     -21.6     1.00
416.6     -22.3     0.95
410.9     -22.9     0.90
405.4     -23.6     0.85
399.8     -24.3     0.80
394.4     -24.9     0.75
389.0     -25.6     0.71
383.6     -26.3     0.67
378.4     -27.0     0.63
373.1     -27.6     0.59
368.0     -28.3     0.56
362.9     -29.0     0.53
357.8     -29.6     0.49
352.8     -30.3     0.46
347.9     -31.0     0.44
343.0     -31.7     0.41
338.2     -32.3     0.38
333.4     -33.0     0.36
328.7     -33.7     0.34
324.0     -34.4     0.31
319.4     -35.0     0.29
314.9     -35.7     0.27
310.4     -36.4     0.26
305.9     -37.1     0.24
301.5     -37.7     0.22
297.2     -38.4     0.21
292.9     -39.1     0.19
288.6     -39.8     0.18
284.4     -40.4     0.17
280.3     -41.1     0.16
276.2     -41.8     0.14
272.1     -42.5     0.13
268.1     -43.2     0.12
264.1     -43.8     0.11
260.2     -44.5     0.11
256.3     -45.2     0.10
252.5     -45.9     0.09
248.8     -46.6     0.08
245.0     -47.3     0.08
241.3     -47.9     0.07
237.7     -48.6     0.06
234.1     -49.3     0.06
230.6     -50.0     0.05
227.0     -50.7     0.05
223.6     -51.4     0.05
220.2     -52.1     0.04
216.8     -52.7     0.04
213.4     -53.4     0.03
210.1     -54.1     0.03
206.9     -54.8     0.03
203.7     -55.5     0.03
200.5     -55.8     0.02
197.4     -55.8     0.03
194.3     -55.8     0.03
191.2     -55.8     0.03
188.3     -55.8     0.03
185.3     -55.7     0.03
182.4     -55.7     0.03
179.6     -55.7     0.03
176.8     -55.7     0.03
174.0     -55.7     0.03
171.3     -55.6     0.03
168.6     -55.6     0.03
166.0     -55.6     0.03
163.4     -55.6     0.03
160.9     -55.5     0.03
158.4     -55.5     0.03
155.9     -55.5     0.03
153.5     -55.5     0.03
151.1     -55.5     0.03
148.7     -55.4     0.03
146.4     -55.4     0.04
144.1     -55.4     0.04
141.9     -55.4     0.04
139.7     -55.4     0.04
137.5     -55.3     0.04
135.4     -55.3     0.04
133.3     -55.3     0.04
131.2     -55.3     0.04
129.1     -55.2     0.04
127.1     -55.2     0.04
125.2     -55.2     0.04
123.2     -55.2     0.04
121.3     -55.2     0.04
119.4     -55.1     0.05
117.6     -55.1     0.05
115.7     -55.1     0.05
113.9     -55.1     0.05
112.2     -55.0     0.05
110.4     -55.0     0.05
108.7     -55.0     0.05
107.0     -55.0     0.05
105.3     -54.9     0.05
103.7     -54.9     0.05
102.1     -54.9     0.05
100.5     -54.9     0.06
98.9     -54.8     0.06
97.4     -54.8     0.06
95.9     -54.8     0.06
94.4     -54.8     0.06
92.9     -54.7     0.06
91.5     -54.7     0.06
90.1     -54.7     0.06
88.7     -54.7     0.06
87.3     -54.6     0.07
85.9     -54.6     0.07
84.6     -54.6     0.07
83.3     -54.6     0.07
82.0     -54.5     0.07
80.7     -54.5     0.07
79.5     -54.5     0.07
