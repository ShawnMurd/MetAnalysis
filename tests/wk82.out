 Virtual and density temperatures of air parcels lifted from lowest level
  
p (mb)  Tv (reversible)  Trho (reversible) Tv (pseudoadiabatic)  Trho (pseudoadiabatic)
  994.4          28.91          28.91              28.91              28.91
  983.2          27.94          27.94              27.94              27.94
  972.1          26.96          26.96              26.96              26.96
  961.1          25.99          25.99              25.99              25.99
  950.1          25.01          25.01              25.01              25.01
  939.3          24.04          24.04              24.04              24.04
  928.6          23.07          23.07              23.07              23.07
  918.0          22.10          22.10              22.10              22.10
  907.4          21.12          21.12              21.12              21.12
  897.0          20.16          20.16              20.16              20.16
  886.7          19.44          19.41              19.44              19.44
  876.4          18.97          18.87              18.96              18.96
  866.3          18.49          18.34              18.49              18.49
  856.2          18.02          17.80              18.01              18.01
  846.2          17.54          17.26              17.54              17.54
  836.3          17.06          16.71              17.05              17.05
  826.5          16.57          16.17              16.57              16.57
  816.8          16.09          15.62              16.08              16.08
  
  
                    ALL AREAS IN UNITS OF J/kg


 Origin    Rev.    P.A.    Rev.    P.A.    Rev.    P.A.
 p (mb)     PA      PA      NA      NA     CAPE    CAPE    DCAPE
 ------    ----    ----    ----    ----    ----    ----    -----
  994.4     0.0     1.0     0.0    45.8     0.0   -44.9     0.0
  983.2     0.0     1.3    44.0    39.8   -44.0   -38.4    17.0
  972.1     0.0     1.7    37.8    34.3   -37.8   -32.6    32.2
  961.1     0.2     2.1    32.2    29.2   -32.0   -27.0    45.5
  950.1     0.5     2.6    26.8    24.2   -26.2   -21.6    57.1
  939.3     0.8     3.0    22.1    20.0   -21.4   -17.0    66.8
  928.6     1.5     3.9    15.9    14.4   -14.4   -10.5    74.3
  918.0     2.0     4.5    12.5    11.3   -10.5    -6.8    80.5
  907.4     3.0     5.5     7.6     6.9    -4.6    -1.3    84.3
  897.0     3.7     6.1     5.3     4.7    -1.6     1.4    87.1
  886.7     4.9     7.4     2.6     2.4     2.3     5.0    87.3
  876.4     6.5     8.7     0.5     0.4     6.0     8.3    85.6
  866.3     5.4     7.2     0.2     0.2     5.2     7.1    90.3
  856.2     2.2     3.1     0.4     0.3     1.8     2.8   107.6
  846.2     0.5     0.8     0.6     0.5    -0.0     0.4   125.3
  836.3     0.0     0.0     0.0     0.6     0.0    -0.6   144.3
  826.5     0.0     0.0     0.0     0.0     0.0     0.0   164.0
  816.8     0.0     0.0     0.0     0.0     0.0     0.0   184.3
